### PR TITLE
enable LMS insecure connections

### PR DESCRIPTION
**Pull request by taroface**
[DOC-10384](https://cockroachlabs.atlassian.net/browse/DOC-10384)

### CODE REVIEW

**Review by ryanluu12345 — APPROVED** [review]
LGTM, let's merge and close this out.

File: src/current/v23.2/live-migration-service.md [hunk 1]
```diff
@@ -135,6 +135,25 @@ lms:
...
 `lms.replicaCount` determines the number of LMS instances created as `lms` pods on the Kubernetes cluster, across which application traffic is distributed. This defaults to `3`.
 
+#### LMS connection
+
+~~~ yaml
+lms:
+  allowTLSDisable: false
+~~~
+
+`lms.allowTLSDisable` specifies whether insecure LMS connections to databases are allowed. This should be set to `true` **only** if secure SSL/TLS connections to the source or target database are not possible. [Secure SSL/TLS connections](#security) should be used by default.
```
Comment (mdlinville) [inline src/current/v23.2/live-migration-service.md:145]:
```suggestion
`lms.allowTLSDisable` enables insecure LMS connections to databases, and is disabled by default. Enable insecure connections **only** if a secure SSL/TLS connections to the source or target database is not possible. When possible, [secure SSL/TLS connections](#security) should be used.
```
Completely optional, just trying out streamlining this a bit.

File: src/current/v23.2/live-migration-service.md [hunk 2]
```diff
@@ -135,6 +135,25 @@ lms:
...
+
+{{site.data.alerts.callout_success}}
+You can also set the `--allow-tls-mode-disable` [flag](#global-flags) to enable insecure LMS connections.
```
Comment (mdlinville) [inline src/current/v23.2/live-migration-service.md:148]:
```suggestion
 The `--allow-tls-mode-disable` [flag](#global-flags) enables insecure LMS connections by setting `lms.allowTLSDisable` to `true`.
```
Is this how it works?

**Review by mdlinville — APPROVED** [review]
Couple of completely optional suggestions. LGTM!

**Pull request merged**